Curtains/Curtains.Infrastructure/Migrations/CurtainsContextModelSnapshot.cs

```diff
@@ -0,0 +1,700 @@
+﻿// <auto-generated />
+using System;
+using Curtains.Infrastructure.Models;
```

Comment:
There seems to be an issue, maybe we should change this dependency to using Curtains.Domain.Models;